framework/jaif-inference-scripts/insert-annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
I think the best thing to do is for the insert-annotations-to-source to be able to receive a set of .jaif and .java files. But there seems to be a challenge if we do so: If you pass a set of .jaif files and a set of .java files to insert-annotations-to-source, how would it correctly associate a .java file to its respective .jaif file? The script infer-and-annotate.sh receives as input a root folder instead of all .java files, so that it can use the relative path of a file on that folder to find its respective .jaif file. This is the answer to this comment: https://github.com/typetools/checker-framework/pull/477#discussion_r42325949